File: Keepr/Models/Keep.cs
namespace Keepr.Models;

public class Keep : RepoItem<int>
{
    public string Name { get; set; }
    public string Description { get; set; }
    public string Img { get; set; }
    public int Views { get; set; } = 0;
    public int Kept { get; set; } = 0;
    public string CreatorId { get; set; }
    public Profile Creator { get; set; }
}

File: Keepr.client/src/models/Keep.js
import { Profile } from "./Account.js"



export class Keep {
    constructor(data) {
        this.id = data.id
        this.name = data.name
        this.description = data.description
        this.img = data.img
        this.views = data.views
        this.kept = data.kept
        this.creator = new Profile(data.creator)
        this.creatorId = data.creatorId
    }
}